contracts/MemeBase.sol

```diff
@@ -157,34 +158,41 @@ contract MemeBase is MemeFactory {
     function _redemptionSetup(address[] memory accounts, uint256[] memory amounts) private {
         require(accounts.length == amounts.length);
 
-        redemptionAddress = address(new Meme("Agent Token", "AGENT", DECIMALS, MIN_TOTAL_SUPPLY));
+        redemptionAddress = address(new Meme("Agent Token II", "AGNT II", DECIMALS, 1_000_000_000 ether));
```

Comment:
This needed fixing. Shouldn't use the same ticker as this will confuse. Also supply was wrong. Please double check